src/models/account/index.js

```diff
@@ -0,0 +1,3 @@
+export { authReducer } from './reducer.js';
+export { authWatcher, login, registretion, logOut, fetchUsersList, fetchUnAssignedUserList } from './sagas.js';
```

Comment:
Do we really need to export all these sagas? I think that we should export only main watcher

Reply:
Yes, I have errors if I export only authWatcher